### PR TITLE
feat(form): allow forge form to be customised in terms of tag and class

### PR DESCRIPTION
**Pull request by TomSmith27**
…sses

### CODE REVIEW

File: packages/ui/src/components/forms/form/Form.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
   <b-form novalidate class="position-relative shadow-sm border p-4 bg-white" @submit.prevent="submit">
     <template v-if="showTitle">
-      <h3 data-cy="form-title">
+      <component :is="titleTag" data-cy="form-title" :class="titleClass">
```
Comment (Polymeta) [inline packages/ui/src/components/forms/form/Form.vue:4]: this only supports html tags though, right? as others would need to get imported somehow

Comment (TomSmith27) [reply packages/ui/src/components/forms/form/Form.vue:4]: ye true you couldnt do a custom component but thinking about this now i think a slot will work better so will update it to use that

Comment (Polymeta) [reply packages/ui/src/components/forms/form/Form.vue:4]: oh yeah, slot would be best for that

File: packages/ui/src/components/forms/form/Form.vue [hunk 2]
```diff
@@ -65,6 +65,10 @@ const props = defineProps({
   loadingText: {
     type: String,
     default: ""
+  },
+  titleClass: {
```
Comment (Polymeta) [inline packages/ui/src/components/forms/form/Form.vue:69]: do we still want to hold onto that even with the slot?

Comment (TomSmith27) [reply packages/ui/src/components/forms/form/Form.vue:69]: I think its still useful cause if you just want to change the text color for example you don't have to make the whole slot can just add a little class

Comment (Polymeta) [reply packages/ui/src/components/forms/form/Form.vue:69]: ah fair, makes sense

**Review by Polymeta — APPROVED** [review]

**Pull request merged**